src/Commands/BannedKeywordsBag.ts

```diff
@@ -0,0 +1,204 @@
+            "internal": true,
+            "is": true,
+            "lambda": true,
+            "let": true,
+            "lock": true,
+            "log": true,
+            "log10": true,
+            "long": true,
+            "module": true,
+            "namespace": true,
+            "native": true,
+            "new": true,
+            "next": true,
+            "nil": true,
+            "not": true,
+            "null": true,
+            "number": true,
+            "object": true,
+            "of": true,
+            "open": true,
+            "operator": true,
+            "or": true,
+            "out": true,
+            "override": true,
+            "package": true,
+            "params": true,
+            "partial": true,
+            "pass": true,
+            "pi": true,
+            "print": true,
+            "private": true,
+            "protected": true,
+            "public": true,
+            "raise": true,
+            "range": true,
+            "readonly": true,
+            "redo": true,
+            "ref": true,
+            "remove": true,
+            "require": true,
+            "rescue": true,
+            "retry": true,
+            "return": true,
+            "sbyte": true,
+            "sealed": true,
+            "self": true,
+            "set": true,
+            "short": true,
+            "sin": true,
+            "sizeof": true,
+            "sqrt": true,
+            "stackalloc": true,
+            "static": true,
+            "strictfp": true,
+            "string": true,
+            "struct": true,
+            "super": true,
+            "switch": true,
+            "symbol": true,
+            "synchronized": true,
+            "tan": true,
+            "then": true,
+            "this": true,
+            "throw": true,
+            "throwas": true,
+            "throws": true,
+            "transient": true,
+            "true": true,
+            "try": true,
+            "type": true,
+            "typeof": true,
+            "uint": true,
+            "ulong": true,
+            "unchecked": true,
+            "undef": true,
+            "unless": true,
+            "unsafe": true,
+            "until": true,
+            "ushort": true,
+            "using": true,
+            "value": true,
+            "var": true,
+            "virtual": true,
+            "void": true,
+            "volatile": true,
+            "when": true,
+            "where": true,
+            "while": true,
+            "with": true,
+            "write": true,
+            "yield": true,
+            "zeros": true,
+        };
+    }
+
+    /**
+     * Returns whether the variable name is allowed.
+     * 
+     * @param name  The variable name.
+     * @returns A boolean determining the validity of the variable name.
+     */
+    public validateName(alias: string): boolean {
+        if (this.bannedKeywords[alias]) {
+            throw new Error(`Invalid variable name requested: '${alias}'.`);
```

Comment:
In general, throwing an error should only happen if something horrible and unexpected has happened. Something being a banned keyword is completely expected. You should return a `true`/`false` boolean as stated in your method description, instead of either returning `false` or throwing an error.

Someone using this class shouldn't have to happen to know that it can throw an error. Wrapping usage in `try`/`catch` blocks is really messy. They should instead choose to deal with the return value as they want.